pkg/http/core/kubernetes/deploy.go

```diff
@@ -134,18 +133,19 @@ func Deploy(c *gin.Context, dm DeployManifestRequest) {
 			return
 		}
 
-		err = kc.VersionVolumes(u, dm.RequiredArtifacts)
+		namespace := u.GetNamespace()
+		err = kc.VersionVolumes(u, namespace, application, client)
+
 		if err != nil {
```

Comment:
Let's put defined errors next to their error checks.
```go
		namespace := u.GetNamespace()

		err = kc.VersionVolumes(u, namespace, application, client)
		if err != nil {
```

Reply:
done